test-packages/test-utils/src/utils/semantic-model-provider.ts

```diff
@@ -99,3 +213,71 @@ export function generateModel(
   }
   return model;
 }
+
+// Fix functions for model issues unfixable by schema changes
+type LibraryFix = (content: Json) => void;
```

Comment:
how do we handle this in none test flows?

Reply:
We don't, but in productive code `strict` is false so all these issues are ignored.
I'll add tests for this.

Reply:
Do you think we should run all the tests in non-strict mode and without these fixes?

Reply:
Hmm, running tests in similar way to productive flows is generally a good things